Sources/SwifterSwift/UIKit/UILabelExtensions.swift

```diff
@@ -34,6 +34,40 @@ public extension UILabel {
         label.sizeToFit()
         return label.frame.height
     }
+    
+    
+    /// SwifterSwift: Add attribute to UILabel
+    /// - Parameters:
+    ///   - range: The range of the string to apply attributes
+    func attribute(textIn range: Range<String.Index>?, _ attributes: [NSAttributedString.Key : Any]) {
+        guard let range = range else {
```

Comment:
If these values can't be `nil` then why make them optional?

Reply:
Made those optional so the developer does't have to unwrap or check with conditions. 
Remove optional? Sure, I'll do that. Let me know.